client/src/main/java/com/vesoft/nebula/client/graph/NebulaGraphConnection.java

```diff
@@ -0,0 +1,56 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula.client.graph;
+
+
+import com.facebook.thrift.TException;
+
+
+/**
+ * @author huangzhaolai-jk
+ * @version 1.0.0
+ * @Description NebulaGraphConnection
+ * @Date 2020/3/17 - 14:00
+ */
+public interface NebulaGraphConnection {
```

Comment:
This is similar with `GraphClient`?

Reply:
yes，you are right - =。Because i used it out of nebula-client jar ever，and made some changes at this class for my program，i forgot it。In fact, I can remove the NebulaGraphConnection.java here and use GraphClient。